frontend/src/views/Pages/ResetPassword.vue

```diff
@@ -139,8 +142,15 @@ export default {
       return { valid: false, errors }
     },
   },
-  created() {
-    //console.log('resetpage', this.$route)
+  async created() {
+    const optin = this.$route.params.optin
+    const result = await loginAPI.loginViaEmailVerificationCode(optin)
+    if (result.success) {
+      this.authenticated = true
+      this.session_id = result.result.data.session_id
+    } else {
+      alert(result.result.message)
```

Comment:
We need a proper error handling here!

Reply:
Can we solve this in general for the whole application? (Am not a fan of toasts, since they are so far away from the vision center & non-oersisten)